llmstudio/engine/providers/azure.py

```diff
@@ -23,7 +33,7 @@ class AzureRequest(ChatRequest):
     api_version: Optional[str] = None
     base_url: Optional[str] = None
     parameters: Optional[AzureParameters] = AzureParameters()
-    functions: Optional[List[Dict[str, Any]]] = None
+    tools: Optional[List[Dict[str, Any]]] = None
```

Comment:
this requires changes on the clients using function calling with azure openai models currently, right?

Reply:
You're absolutely right. I'm currently working on a fix to revert that change.